frontend/src/containers/CreateRecipe/CreateRecipe.css

```diff
@@ -0,0 +1,12 @@
+.createRecipePage.container-fluid {
+    margin: 0px;
+    padding-top: 50px;
+    width: 100%;
+}
+
+
+.edit-recipe-form {
+    margin: 0px;
+    margin-top: 100px;
+    width: 100%;
+}
```

Comment:
NP: Let's ensure that every file has a trailing newline. There's a vscode setting which will add trailing newlines for you automatically on-save.